### DISCUSSION TITLE
Can renovate block packages which are not in approved list?

### DISCUSSION THREAD

**Discussion by marunmohan** [renovatebot/renovate#25755] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

Azure DevOps (dev.azure.com)

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Hi,

We have our organization approved nuget and npm packages. I want to know whether Renovate can block or report any packages used apart from approved nuget and npm packages? Also, can renovate update only the packages which are approved from the organization prespective?

### What you tried so far.

I haven't tried anything so far

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
Renovate does not have any blocking or reporting capabilities, but you definitely can use packageRules to choose which packages to update or not

**Reply by marunmohan** [reply]
Thanks for the response.
